nvme-print-stdout.c

```diff
@@ -5973,6 +5973,63 @@ static void stdout_pull_model_ddc_req_log(struct nvme_pull_model_ddc_req_log *lo
 	d((unsigned char *)log->osp, osp_len, 16, 1);
 }
 
+static void stdout_relatives(const char *name, nvme_root_t r)
+{
+	int id;
+	int nsid;
+	int ret = sscanf(name, "nvme%dn%d", &id, &nsid);
```

Comment:
Please avoid mixing declaration and assignment. Also order the variables so we get an inverse Christmas tree style.